src/test/scala/com/ibm/couchdb/api/DocumentsSpec.scala

```diff
@@ -132,6 +132,21 @@ class DocumentsSpec extends CouchDbSpecification {
       docs.getDocsData mustEqual Seq(fixAlice, fixAlice)
     }
 
+    "Get all documents by type and include the doc data" >> {
```

Comment:
Could you please also add a test for the other queryByTypeIncludeDocs? It's not covered right now.